docs/elements/demo.html

```diff
@@ -36,9 +36,9 @@
     <script async src="https://ga.jspm.io/npm:es-module-shims@1.8.1/dist/es-module-shims.js"></script>
   </head>
   <body unresolved>{% if demo.filePath %}
-    <main>
+    <rh-surface role="main">
```

Comment:
I don't think this surface is necessary here. I believe all the necessary CSS is already provided by `/styles/demo/styles.css`